packages/spawner/scripts/build-bundle-spawn.js

```diff
@@ -0,0 +1,33 @@
+/* global __dirname */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'ses';
+import fs from 'fs';
+import process from 'process';
+import bundleSource from '@agoric/bundle-source';
+
+async function writeSourceBundle(contractFilename, outputPath) {
+  await bundleSource(contractFilename).then(bundle => {
+    // TODO: fix
+    // @ts-ignore mkdirSync believes it only accepts 2 arguments.
+    fs.mkdirSync(`${__dirname}/../bundles`, { recursive: true }, err => {
+      if (err) throw err;
+    });
+    fs.writeFileSync(outputPath, `export default ${JSON.stringify(bundle)};`);
+  });
+}
+
+async function main() {
+  const contractFilename = `${__dirname}/../src/vat-spawned.js`;
+  const outputPath = `${__dirname}/../bundles/bundle-spawn.js`;
+  await writeSourceBundle(contractFilename, outputPath);
```

Comment:
(Not an issue for this PR)

That seems like a security hole: `dirname` is a global, and you can navigate upwards from it?  Is this not available from SES?